crates/data/src/solana/tables/instruction.rs

```diff
@@ -29,6 +35,7 @@ table_builder! {
         a14: Base58Builder,
         a15: Base58Builder,
         rest_accounts: AccountListBuilder,
+        accounts_bloom: FixedSizeBinaryBuilder = FixedSizeBinaryBuilder::new(64, 0),
```

Comment:
@eldargab is it good enough?

Reply:
:fire:

Reply:
Can the `Default` implementation be removed completely now?

Reply:
i would be happy but `table_builder` macros requires every field to implement `Default`